Apps/WebClient/src/ClientApp/src/store/modules/immunization/actions.ts

```diff
@@ -18,12 +18,15 @@ export const actions: ImmunizationActions = {
         );
 
         return new Promise((resolve, reject) => {
-            if (context.state.status === LoadStatus.LOADED) {
+            if (
+                context.state.immunizations[params.hdid]?.status ===
+                LoadStatus.LOADED
```

Comment:
I suggest swapping to use `getImmunizationDatasetState(context.state, params.hdid)` as it is slightly safer in general and that's what we're doing in the other modules now.

Reply:
Fixed.

Reply:
Fixed